src/content/docs/mobile-monitoring/new-relic-mobile/get-started/introduction-app-launch-times.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Introduction to app launch times
+tags:
+  - Mobile monitoring
+  - New Relic Mobile
+  - Get started
+metaDescription: "Use New Relic's mobile monitoring capabilities to manage the performance of your iOS and Android apps and your customers' overall digital experience."
+---
+
+New Relic's mobile monitoring capabilities help you gain deeper visibility into how to analyze your [Android](/docs/mobile-monitoring/new-relic-mobile-android/get-started/introduction-new-relic-mobile-android) and [iOS](/docs/mobile-monitoring/new-relic-mobile-ios/get-started/introduction-new-relic-mobile-ios) application performance and troubleshoot crashes. With the abundance of high-quality apps out there, users have become very sensitive to app launch time and have very high expectations.
+
+## Why would you track app launch time [#features]
+
+Speed is an essential part of app performance. Users expect apps to be responsive and fast to load. An app with a slow start time doesn’t meet this expectation can be disappointing to users. App launch speed is very indicative of the overall quality of your app and tracking it will help assess the responsiveness of your apps. New Relic provides the ability to helps you track key metrics to improve your app’s performance, including cold time, hot time and more.
+
+* Cold start
+A cold start refers to an app’s starting from scratch. This means that the app’s process has not been created by the system until then. Cold starts occur when you are launching the app for the first time since booting, or if the app was killed. Since there is no existing process up and running this is the slowest type of launch. Cold starts are usually the main focus when it comes to minimizing app launch time due to them taking the longest.
+
+* Hot start
+A hot start refers to when your app’s process is already running in the background. In a hot start, all the system has to do is bring your app to the foreground. This makes it a lot faster than a cold start because the app is already present in memory. The system doesn’t have to repeat the initialization so it has much less overhead.
+
+To the user, there is very little difference between a cold start scenario and a hot start, which is why it’s important to minimize the time taken in both scenarios.
+
+## Mobile app launch time benchmarks
+According to Google, your app’s startup time is not considered excessive if they fall within these thresholds:
+
+* Cold startup should take less than 5 seconds
+* Hot startup should take less than 1.5 seconds
+
+According to Apple, your app’s startup time is not considered excessive if they fall within 400ms. The app must launch in less than 2o seconds or the system will kill the app.
+
+
+## Configurations
+You'll be able to configure mobile app launch times for [Android](docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-app-launch-time-android-apps)and iOS.
```

Comment:
@ywang-nr how do iOS users configure launch time?

Reply:
@rhetoric101 Thanks for the edits, they look great and I made one adjustment. IOS is not released yet, so we don't have a doc yet, can we add that later when it's ready?